### PR TITLE
Json1 presence

### PR DESCRIPTION
**Pull request by curran**
Big step closer to #16.

This PR makes [ot-json1-presence](https://github.com/vizhub-core/json1-presence), a fresh fork of `json1-presence`, available to both the frontend code and the backend code. This also necessitated upgrades to related packages [sharedb-client-browser](https://github.com/vizhub-core/sharedb-client-browser) and [codemirror-ot](https://github.com/vizhub-core/codemirror-ot).

### CODE REVIEW

**Comment by curran** [comment]
Status: The JSON1 with presence part is bundling correctly, but now there's an issue with codemirror-ot:

https://github.com/vizhub-core/codemirror-ot/issues/36

Will address that then come back to this.

File: test/app.jsx [hunk 1]
```diff
@@ -1,2 +1,2 @@
 const foo = <div></div>;
-const foo2 = hi;
+const foosdfd2d = hi;
```
Comment (curran) [inline test/app.jsx:2]:
```suggestion
const foo2 = hi;
```

File: test/script.ts [hunk 2]
```diff
@@ -1 +1 @@
-const testing = 'Hello World!';
+const tesdfdsting = 'Hello World!';
```
Comment (curran) [inline test/script.ts:1]:
```suggestion
const testing = 'Hello World!';
```

**Comment by curran** [comment]
@Anooj-Pai This is ready for review. There is no behavior change, but now we should have access to use the ShareDB Presence API with JSON1, in theory. Please make sure it builds and runs OK on your system. There were a lot of build-related changes upstream. Thanks!

**Review by Anooj-Pai — APPROVED** [review]
This looks great! Everything builds and run perfectly and I will add the new sidebar component ASAP

**Comment by curran** [comment]
Excellent! Thank you.

**Pull request merged**